code/modules/events/blob.dm

```diff
@@ -1,7 +1,8 @@
 /datum/round_event_control/blob
 	name = "Blob"
 	typepath = /datum/round_event/ghost_role/blob
-	weight = 10
+	//weight = 10 //ORIGINAL
+	weight = 5 //SKYRAT EDIT CHANGE
```

Comment:
Blob is already so rare, I barely see it once a day :(

Reply:
Aw, I like blob. That's a shame.

Reply:
I dont mind having it be either, convince Xyel as he wants to have it changed

Reply:
No longer reduced